org.flatpak.gnomeConnectionManager.yml

```diff
@@ -0,0 +1,59 @@
+app-id: org.flatpak.gnomeConnectionManager
+runtime: org.gnome.Platform
+  #runtime-version: '3.38'
+runtime-version: '43'
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --filesystem=home
+sdk: org.gnome.Sdk
+command: /app/bin/gnome_connection_manager.py
+modules:
+  - name: vte
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/vte/0.72/vte-0.72.1.tar.xz
+        sha256: 0554f9f88d56ce2d78398fcc7f69bc00e53bbbc5f694e0ae1dcaf5286f89d7e4
+  - name: gtk
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gtk+/3.24/gtk%2B-3.24.37.tar.xz
+        sha256: 6745f0b4c053794151fd0f0e2474b077cccff5f83e9dd1bf3d39fe9fe5fb7f57
+  - name: libcanberra
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: http://0pointer.de/lennart/projects/libcanberra/libcanberra-0.30.tar.xz
+        md5: 34cb7e4430afaf6f447c4ebdb9b42072
+  - name: tcl
+    buildsystem: autotools
+    subdir: unix
+    post-install:
+      - chmod 755 /app/lib/libtcl8.6.so
+      - make install-private-headers
+    build-options:
+      config-opts:
+        - --enable-framework=yes
+    sources:
+      - type: archive
+        url: http://downloads.sourceforge.net/sourceforge/tcl/tcl-core8.6.12-src.tar.gz
+        md5: 0802e5c9e8936fc9b0c69a8da4cedfe2
+  - name: expect
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: http://downloads.sourceforge.net/expect/expect5.45.tar.gz
+        md5: 44e1a4f4c877e9ddc5a542dfa7ecc92b
+  - name: gcm
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/yma-het/gnome-connection-manager.git
+        commit: 9886cc2f8cae36ba176885db26ffe555ae27b306
+    build-commands:
+      - mkdir -p /app/bin
+      - cp -r * /app/bin
+      - chmod +x /app/bin/gnome_connection_manager.py
```

Comment:
you need to install the .desktop and icons in the right place. The former also needs to be edited using `desktop-file-edit` .

Also, I don't see an apptream file.

Reply:
Added appstream file. Installed desktop file in the right place. Icon path is already matches .desktop file.